src/__tests__/Layout-test.js

```diff
@@ -14,6 +14,22 @@ var texts = layoutTestUtils.texts;
 var textSizes = layoutTestUtils.textSizes;
 
 describe('Layout', function() {
+
+  it('should accomodate nodes that already have layout and children properties', function() {
```

Comment:
The way I would fix this is to add another describe, and when we do the transpiling of the tests, only transpile the tests that are in this describe.